pycheribuild/config/target_info.py

```diff
@@ -567,6 +567,61 @@ def get_rootfs_target(self) -> "Project":
         from ..projects.cross.newlib_baremetal import BuildNewlibBaremetal
         return BuildNewlibBaremetal.get_instance(self.project)
 
+class NewlibRtemsTargetInfo(_ClangBasedTargetInfo):
+    shortname = "Newlib RTEMS"
+
+    @property
+    def sdk_root_dir(self) -> Path:
+        return self.config.cheri_sdk_dir
+
+    @property
+    def sysroot_dir(self) -> Path:
+        # Install to target triple as RTEMS' LLVM/Clang Driver expects
+        return self.config.cheri_sdk_dir
+
+    @property
+    def must_link_statically(self):
+        return True  # only static linking works
+
+    @property
+    def _compiler_dir(self) -> Path:
+        return self.config.cheri_sdk_bindir
+
+    @classmethod
+    def toolchain_targets(cls, target: "CrossCompileTarget", config: "CheriConfig") -> typing.List[str]:
+        return ["llvm-native"]
+
+    @property
+    def target_triple(self):
+        return "riscv64-unknown-rtems5"
+
+    @classmethod
+    def base_sysroot_targets(cls, target: "CrossCompileTarget", config: "CheriConfig") -> typing.List[str]:
+        return ["newlib", "compiler-rt-builtins"]
+
+    def required_compile_flags(self) -> typing.List[str]:
+        return [""]
+
+    @property
+    def local_install_root(self) -> Path:
+        return self.config.cheri_sdk_dir
+
+    @property
+    def additional_executable_link_flags(self):
+        """Additional linker flags that need to be passed when building an executable (e.g. custom linker script)"""
+        return [""]
+
+    @property
+    def is_baremetal(self):
+        return False
+
+    @property
+    def is_rtems(self):
+        return True
```

Comment:
A default implementation returning false is needed in the base classes (see jenkins failure)